lib/fastlane/actions/unlock_keychain.rb

```diff
@@ -31,12 +39,24 @@ def self.add_keychain_to_search_list(keychain_path)
         unless keychains.include?(keychain_path)
           keychains << keychain_path
 
-          commands = []
-          commands << Fastlane::Actions.sh("security list-keychains -s #{keychains.shelljoin}", log: false)
-          commands
+          Fastlane::Actions.sh("security list-keychains -s #{keychains.shelljoin}", log: false)
+          if Helper.is_test?
+            "security list-keychains -s <add>"
```

Comment:
hey @bartoszj - maybe you could print out a few statements that would show what _would_ happen if this were to be executed fully?

also, it will be good to move line 42 down below this if clause to ensure it gets returned.